vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -732,6 +817,14 @@
             		$('.modal-footer').append(alert); 
               	});
             	
+
+	            $(document).on("click", ".deleteSpace", function(e) {
+		            $("#confirmDeleteSpaceAlert").show();
+		            var alert = $('<input type="hidden" id="deleteSpaceId">');
+		            $(alert).attr( 'value', $(this).siblings('input').val());
+		            $('.modal-footer').append(alert);
+	            });
+
```

Comment:
indentation

Reply:
this looks right? its just new code

Reply:
![image](https://github.com/user-attachments/assets/8c279359-41a5-4bb3-9857-1d544c171d9f)

Reply:
<img width="821" alt="Screenshot 2024-09-13 at 12 53 01" src="https://github.com/user-attachments/assets/999b96d8-c641-4cf3-b49f-f287d5095b22">
this is how it looks on my side, i don't see anything

Reply:
you probably have tabs and spaces differently than the rest